### ISSUE TITLE
Remote Repair Tool zeroed dex??

### ISSUE THREAD

**Issue by cptYokom** [NCC-Lykos/neocron-classic#192]
Was out in the rhino tank killing warbots. I used the creeper glove with ppu to repair the tank with remote repair tool. 

Boom my dex levled to ZERO. I have relogged a few times and genrepped. Still zero dex.  Not sure how to fix this

![zerodex](https://user-images.githubusercontent.com/36430896/36645787-cd962a08-1a65-11e8-9f12-746b9ca33fa3.png)

**Comment by cptYokom** [comment]
This char name is Wanheda if you were inclined to go in and fix the dex.

**Comment by cptYokom** [comment]
Its been 3 days no action on this, I know its beta and all but now I'm scared to use the remote repair tool, and not to mention my wife's monk is still missing 20 levels of dex.

**Comment by ghost** [comment]
Sorry we all have full time jobs?

I'm not trying to be an asshole or anything, but all of us on the team work full-time.  I'm sending this over to @NCC-Therenall though to see if we may be able to expedite this.

**Comment by cptYokom** [comment]
I understand, so I did some more test. I made a spy and took him out to try and repair the tank with the tool with the repair glove. First time it did the same zeroed his dex. So I made a new one and this time it zeroed more then just his dex. It zeroed 3 of his stats, Int Str and Dex. Now the tank did get repaired, and with drugs for dex he can still repair the tank. 

// FYI ya the tool came out looking like a pistol even after I zoned twice

Ill try more test with the TL 80 remote repair tool 
![repair1](https://user-images.githubusercontent.com/36430896/36936917-7e27fb70-1f03-11e8-8b61-843961b013cc.png)
![repair2](https://user-images.githubusercontent.com/36430896/36936919-81790d00-1f03-11e8-987a-0507cdae658d.png)
